### PR TITLE
Fix "fill max" for mobile

### LINKED ISSUE #322: [BUG] there is not max button for items in bazzar
after updating the extension to version 5.4 i can't see any max on bazaar items that i can select to fill the number of items buying to max available.

### PR DESCRIPTION
**Pull request by Sashank999**
Closes #322 .

### CODE REVIEW

File: torntools/scripts/content/bazaar/ttBazaar.js [hunk 1]
```diff
@@ -58,43 +58,26 @@ requireDatabase().then(() => {
...
+							doc.find("[class*='buyMenu_'] [class*='amount_']").insertAdjacentHTML("beforeEnd", rawHTML);
+							doc.find("[class*='buyMenu_'] [class*='buyForm_'] input[class*='numberInput_']").addEventListener(
+								"keyup",
+								(event) => (doc.find("span#tt-total-cost").innerHTML = "$" + numberWithCommas(bazaarItemPrice * event.target.value, false))
+							);
+						};
+					};
+				});
+			} else {
+				doc.findAll("[class*='itemDescription__']").forEach((buyMenu) => maxBuy(buyMenu, "[class*='description__'] [class*='amountValue_']"));
```
Comment (DeKleineKobini) [inline torntools/scripts/content/bazaar/ttBazaar.js:78]: I doubt this would work when items load after the first load?

**Review by DeKleineKobini — CHANGES_REQUESTED** [review]
Don't forget the changelog as well.

File: torntools/scripts/content/bazaar/ttBazaar.js (hunk 1 above)
Comment (Sashank999) [reply torntools/scripts/content/bazaar/ttBazaar.js]: Added a MutationObserver for that.

**Review by DeKleineKobini — COMMENTED** [review]
Would it now work for the first batch of items on mobile?

**Comment by Sashank999** [comment]
> Would it now work for the first batch of items on mobile?

It does.

**Review by DeKleineKobini — APPROVED** [review]

**Pull request merged**